src/Calculator.Shared/Views/UnitConverter.xaml

```diff
@@ -200,7 +200,7 @@
 		<Style x:Key="ValueMediumStyle"
                BasedOn="{StaticResource ValueBaseStyle}"
                TargetType="controls:CalculationResult">
-			<Setter Property="MaxFontSize" Value="34"/>
```

Comment:
That was the value set by the original sources ... are we using the right font ?

Reply:
The only reason I changed this is because the medium value of calculator result is higher than that. Unsure about the font itself.